listenbrainz_spark/recommendations/recording/tests/test_models.py

```diff
@@ -73,8 +72,9 @@ def test_get_best_model(self, mock_als_cls, mock_params, mock_tvs):
             estimator=mock_als_cls.return_value,
             estimatorParamMaps=mock.ANY,
             evaluator=mock_evaluator,
-            trainRatio=0.80,
-            collectSubModels=True
+            numFolds=5,
```

Comment:
Can we please document these values? And maybe make constants for them, if we anticipate changing them?

Reply:
Done